public/app/tasks/tasks.css

```diff
@@ -1,10 +1,14 @@
 ﻿.task-highlited {
     background-color: #ffffcc;
-    border-left: 6px solid #ffeb3b!important;
+    border-left: 21px solid #ffeb3b!important;
 }
 .task-default {
-    padding: 4px 16px;
-    border-left: 6px solid #F2F7F9;
+    padding: 4px 0px;
+    border-left: 21px solid #F2F7F9;    
+}
+
+.col-md-10 {
```

Comment:
It's moveton to rewrite libraries classes behaviour. You could rewrite it to inherited styles.